installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc

```diff
@@ -59,12 +59,45 @@ Before you install a cluster on bare metal infrastructure that you provision,
 you must create {op-system} machines for it to use. Follow either the steps
 to use an ISO image or network PXE booting to create the machines.
 
-include::modules/installation-user-infra-machines-iso.adoc[leveloffset=+2]
+There are several methods of configuring RHCOS during ISO and
+PXE installations. These include:
+
+* Kernel arguments: For a PXE install, you can APPEND arguments to the
+kernel of the live installer. For an ISO install, you can interrupt the
+live install boot process to add kernel arguments. In both cases, you can use
+`special coreos-inst*` arguments, to direct the live installer, as well as
+standard installation boot arguments, for turning standard kernel services
+on or off.
+
+* Ignition configs: You need to generate an {product-title} Ignition Config
```

Comment:
I agree with "Ignition config" for capitalization. I don't want to use special markup, since that implies that it is entered literally  somewhere, which I don't think it is.